app/common/directives/embed-control.directive.js

```diff
@@ -0,0 +1,18 @@
+module.exports = EmbedControlDirective;
+
+EmbedControlDirective.$inject = [];
+function EmbedControlDirective() {
+    return {
+        restrict: 'A',
+        controller: EmbedControlController
+    };
+}
+
+EmbedControlController.$inject = ['$scope', '$element', '$attrs', '$rootScope', '_'];
+function EmbedControlController($scope, $element, $attrs, $rootScope, _) {
+    if ($rootScope.globalEmbed && !$attrs.embedOnly) {
```

Comment:
It might be better to simply call this directive `embed-only` and always expect a true or false value.